lib/serverpipe.js

```diff
@@ -1,33 +1,11 @@
 /* eslint-env browser */
-import log from 'loglevel';
-import Url from 'url';
-import {getCsrfHeaderName, getCsrfToken} from './csrfToken';
-
-function isIE() {
-    const ua = window.navigator.userAgent;
-    const msie = ua.indexOf('MSIE ');
-    const ie = (msie > 0 || !!navigator.userAgent.match(/Trident.*rv:11\./));
-    if (ie) {
-        log.debug('Running on MS IE');
-    }
-    return ie;
-}
-
-export function addQsToUrl(url, qs) {
```

Comment:
If and when you release this, can you make it an upgrade-breaking release, as various apps use this function.